tests/gateway/test_client.py

```diff
@@ -16,10 +16,10 @@ def basic_config_dict():
     return {
         "routes": [
             {
-                "name": "completions-gpt4",
+                "name": "completions",
                 "type": "llm/v1/completions",
                 "model": {
-                    "name": "gpt-4",
+                    "name": "text-davinci-003",
```

Comment:
I changed these configs because I actually used my own API Key to get the responses that are used in the mocks through the client query API :)